### PR TITLE
ADDED: AudioDevice Class

### PR DESCRIPTION
**Pull request by rox51124**

### CODE REVIEW

File: src/AudioDevice.hpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+** EPITECH PROJECT, 2022
+** indieStudio
+** File description:
+** AudioDevice
+*/
+
+#ifndef AUDIODEVICE_HPP_
+#define AUDIODEVICE_HPP_
+
+#include <raylib.h>
```
Comment (iDommel) [inline src/AudioDevice.hpp:11]: I think we can avoid that include here and put it in the .cpp file

File: test_functions/TestAudioDevice.cpp [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    // Initialization
+    //--------------------------------------------------------------------------------------
+    const int screenWidth = 800;
+    const int screenHeight = 450;
+
+    SetConfigFlags(FLAG_MSAA_4X_HINT);  // NOTE: Try to enable MSAA 4X
+
+    InitWindow(screenWidth, screenHeight, "raylib [audio] example - module playing (streaming)");
+
+    //InitAudioDevice();                  // Initialize audio device
```
Comment (iDommel) [inline test_functions/TestAudioDevice.cpp:34]: I think you forgot to test your class, or at least it's not used in the test file

**Review by iDommel — CHANGES_REQUESTED** [review]
A few minor changes, and please merge dev into this branch to fix the conflicts

**Review by iDommel — DISMISSED** [review]

File: src/AudioDevice.hpp [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    class AudioDevice {
+        public:
+            ///  @brief Constructor a new audio device
+            AudioDevice();
+            ///  @brief Destruct the audio device:: close the audio device
+            ~AudioDevice();
+            ///  @brief Check if the audio device is playing
+            bool Ready();
+            ///  @brief Set the volume of the audio device
+            void Volume(float volume);
```
Comment (MauxC) [inline src/AudioDevice.hpp:21]: setVolume & isReady would be better names for methods

File: src/exceptions/AudioDeviceError.hpp [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+** File description:
+** AudioDeviceError
+*/
+
+#ifndef AUDIODEVICEERROR_HPP_
+#define AUDIODEVICEERROR_HPP_
+
+#include <string>
+
+class AudioDeviceError {
```
Comment (MauxC) [inline src/exceptions/AudioDeviceError.hpp:13]: Namespace indie is missing

**Review by MauxC — CHANGES_REQUESTED** [review]

**Review by MauxC — APPROVED** [review]

**Review by iDommel — APPROVED** [review]

**Pull request merged**